sbtc/tests/integration/validation.rs

```diff
@@ -380,10 +380,14 @@ fn op_csv_disabled() {
         }],
     };
 
-    let expected = "mandatory-script-verify-flag-failed (Locktime requirement not satisfied)";
+    // In bitcoin-core v25 the message is "non-mandatory-script-verify-flag
+    // (Locktime requirement not satisfied)", but in bitcoin-core v27 the
+    // message is "mandatory-script-verify-flag-failed (Locktime
+    // requirement not satisfied)". We match on the part that is probably
+    // consistent across versions.
     match rpc.send_raw_transaction(&tx3).unwrap_err() {
         BtcRpcError::JsonRpc(JsonRpcError::Rpc(RpcError { code: -26, message, .. }))
-            if message == expected => {}
+            if message.ends_with("(Locktime requirement not satisfied)") => {}
```

Comment:
Do you think we need to test for message? Does bitcoin rpc have multiple messages for the same code? I think elsewhere I just did an `if code == -26` for example (thinking if they make small changes the text at some point)

Reply:
Maybe not. But the issue is that I've observed `code == -26` being returned for quite a few different error responses when submitting "invalid" transactions, such as non-minimal-push in script (and maybe some others that I'm not remembering at the moment). And I'd like to check that we're triggering the expected error and not something else.

But yeah this is really brittle. If it gets any worse then we should nix it. I take it you think it's already reached that point?

Reply:
> But yeah this is really brittle. If it gets any worse then we should nix it. I take it you think it's already reached that point?

Nah, doesn't feel super-important right now, I didn't realize it had different messages for the same code. If it becomes an issue later then we can see if there's a better way imo. But that shouldn't happen unless we tweak the minimum bitcoin version in the compose file anyway.